voodoo_winit/Cargo.toml

```diff
@@ -13,5 +13,6 @@ keywords = ["vulkan", "gpu", "gpgpu", "graphics"]
 categories = ["rendering::graphics-api"]
 
 [dependencies]
-winit = "0.10"
+winit = "0.17.1"
```

Comment:
Perhaps it might be better to leave the version at `0.17`, that way we automatically get any non-breaking updates. What do you think?

Reply:
Agreed, I've pushed the change to 0.17